site/using-weave/awsvpc.md

```diff
@@ -3,25 +3,25 @@ title: Using IP Routing on an Amazon Web Services Virtual Private Cloud
 menu_order: 110
 ---
 
-If you are running your container infrastructure entirely within
-Amazon Web Services (AWS) Elastic Compute Cloud (EC2), then you can
-choose AWS-VPC mode to connect your containers without any overlay, so
-they can operate very close to the full speed of the underlying
-network.
-
-In this mode, Weave Net still manages IP addresses and connects
-containers to the network, but instead of wrapping up each packet and
+If your container infrastructure is running entirely within Amazon Web Services (AWS) 
+Elastic Compute Cloud (EC2), then you can enable AWS-VPC mode with Weave Net. In AWS-VPC mode, 
+containers are networked without using an overlay and allows network speeds 
+close to that of the underlying network.
+
+With AWS-VPC enabled, Weave Net manages IP addresses and connects
+containers to the network as usual, but instead of wrapping each packet and
 sending it to its destination, Weave Net instructs the AWS network
-router which ranges of container IP addresses live on which instance.
+router with the ranges of container IP addresses and the instances 
+on which they live. 
 
 ![Weave Net AWS-VPC Mode](weave-net-awsvpc-1007x438.png)
 
-###Configuring your EC2 instances to use Weave AWS-VPC mode
+###Configuring EC2 Instances to use Weave AWS-VPC Mode
 
-First, your AWS instances need to be given access to change the route
-table.  Give them an IAM Role which has this access; if you have an
-existing role then extend it or create a new one with a policy
-allowing the necessary actions:
+First, your AWS instances need to be given write access to the route
+table.  Give them an IAM Role which has the following access, or if you have an
```

Comment:
This isn't an 'or' - they must give access: one way is to extend an existing role, another way is to create a new role.